exercises/react/example.cr

```diff
@@ -0,0 +1,71 @@
+alias CellValue = Int32
+
+private class Cell
+  @dependents = [] of ComputeCell
+
+  def initialize(initial : CellValue)
+    @value = initial
+  end
+
+  getter :value
+
+  protected getter :dependents
+  protected def propagate; _propagate; end
```

Comment:
Ok, it work if I change `@dependents = [] of ComputeCell` to `@dependents = [] of Cell` which  makes sense I guess, but I'm still surprised why it worked when `Reactor` was there.